src/main/java/com/netcracker/study/general/Game.java

```diff
@@ -1,13 +1,87 @@
 package com.netcracker.study.general;
 
+import com.netcracker.study.objects.entities.Bullet;
+import com.netcracker.study.objects.entities.Player;
+
 import java.io.FileNotFoundException;
+import java.util.Set;
 
 public class Game {
     private GameField gameField;
+    private int numberOfPlayers;
+    private int currentPlayer;
+    private Phase currentPhase;
 
     public Game(String mapFilePath) throws FileNotFoundException {
         MapReader mapReader = new MapReader(mapFilePath);
         this.gameField = mapReader.createGameFieldFromTextFile();
+        this.numberOfPlayers = 0;
```

Comment:
numberOfPlayers should be encapsulated behind the collection which store these players:

Collection<Player> players;

player.size()
player.isEmpty()